karaf/jolokia-opts

```diff
@@ -1,8 +1,5 @@
 #!/bin/sh
```

Comment:
maybe it would make sense to use fish-pepper also for the karaf image ? I wonder whether there is enough overlap with the Java image to justify this. At least the jolokia handling is the same.